### PR TITLE
Update stable 0.3 to 0.3.2

### PR DESCRIPTION
**Pull request by jschwe**
Prepare a new release

### CODE REVIEW

File: RELEASES.md
Comment (ogoffart) [inline RELEASES.md:5]:
```suggestion
- Add new `CRATE_TYPES` flag to `corrosion_import_crate()` to restrict which
```

Comment (jschwe) [reply RELEASES.md]: Thanks!

**Comment by jschwe** [comment]
@ogoffart If you have the time, could you (re-)review and formally approve this PR? I added a branch protection rule for `stable` branches to require one approval by another person before merging (to have an extra pair of eyes).

**Review by ogoffart — APPROVED** [review]
(Thanks for your work on corrosion)

**Pull request merged**
